packages/database/lib/schema.ts

```diff
@@ -39,8 +39,17 @@ function pgEnum<T extends string>(columnName: string, enumObj: Record<any, T>) {
   return varchar(columnName, {enum: Object.values(enumObj) as NonEmptyList<T>});
 }
 
+export const daos = pgTable('daos', {
+  daoAcct: pubkey('dao_acct').primaryKey(),
+  name: varchar('name').notNull(),
+  url: varchar('url').notNull(),
+  // In FaaS, each DAO is tied to its own token which futarchic markets will aim to pomp to the moon
+  mintAcct: pubkey('mint_acct').references(() => tokens.mintAcct).notNull()
```

Comment:
Should we also consider adding base_lot_size here? Not sure if it makes sense to store all DAO configuration data off-chain because the place it really matters in on-chain it seems.

Reply:
Right now the lot size is a column in the markets table